docker/env.sh

```diff
@@ -33,6 +33,7 @@ else
 fi
 export BUILD_ARGS="--build-arg GOARCH=${GOARCH}
 --build-arg REACT_APP_AUTH_MODE=${REACT_APP_AUTH_MODE}
+--build-arg REACT_APP_DISABLE_ANALYTICS=${REACT_APP_DISABLE_ANALYTICS}
```

Comment:
since this is a frontend var that would need to work in the built frontend, we'll also need to update `docker/frontend-entrypoint.py` to update the value.

Reply:
In `frontend-entrypoint.py`, we are replacing the values. For URLs that may be good as they are hard coded. I don't think we need to add `REACT_APP_DISABLE_ANALYTICS` to `frontend-entrypoint.py`